src/node_messaging.cc

```diff
@@ -1308,6 +1398,23 @@ static void MessageChannel(const FunctionCallbackInfo<Value>& args) {
       .Check();
 }
 
+static void BroadcastChannel(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  if (!args.IsConstructCall())
+    return THROW_ERR_CONSTRUCT_CALL_REQUIRED(env);
+
+  CHECK(args[0]->IsString());
+
+  Local<Context> context = args.This()->CreationContext();
+  Context::Scope context_scope(context);
+
+  Utf8Value name(env->isolate(), args[0]);
+  MessagePort* port =
+      MessagePort::New(env, context, nullptr, std::string(*name));
+
+  args.This()->Set(context, env->port_string(), port->object()).Check();
```

Comment:
```suggestion
  USE(args.This()->Set(context, env->port_string(), port->object()));
```

(although I think you could also just return `port->object()` here, `BroadcastChannel` doesn’t need to be a constructor?)